src/lib/core/components/computations/plugins/abundance.tsx

```diff
@@ -24,25 +28,144 @@ function variableDescriptorToString(
 }
 
 export function AbundanceConfiguration(props: ComputationConfigProps) {
-  const [name, setName] = useState('New ranked abundance tool');
-  const [rankingMethod, setRankingMethod] = useState(ABUNDANCE_METHODS[0]);
-  const { computationAppOverview, addNewComputation } = props;
+  const {
+    computationAppOverview,
+    computation,
+    analysisState,
+    visualizationId,
+  } = props;
   const studyMetadata = useStudyMetadata();
+  const { url } = useRouteMatch();
+  const history = useHistory();
   // Include known collection variables in this array.
   const collections = useCollectionVariables(studyMetadata.rootEntity);
   if (collections.length === 0)
     throw new Error('Could not find any collections for this app.');
 
-  const [collectionVariable, setCollectionVariable] = useState(
-    variableDescriptorToString({
-      variableId: collections[0].id,
-      entityId: collections[0].entityId,
-    })
-  );
+  const rankingMethod =
```

Comment:
I don't quite follow this part. From my naiive read it seems like both the getAppsDefaultConfigs hook and this section of the plugin are setting the default configuration?